src/controller/login.controller.ts

```diff
@@ -214,7 +218,11 @@ export class LoginController {
       }
       throw new Error('Token not valid');
     } catch {
-      await this.loginByFaasFactory();
+      try {
+        await this.loginByFaasFactory();
```

Comment:
This is ugly but there is not an option yet. If we don't catch it , the error message won't be shown when you use a command that requires logging like add: domain